hermeznetwork/wallet-ui
# Tracks L1 delayed withdrawal txs to remove them if they fail

### What does this PR does?

Tracks non-instant withdrawals (delayed withdrawals in the wallet) in case they fail to update storage and redux so user can retry. Also refactors checking delayed withdrawals to occur in the home and account-details.

### How to test?

Make a delayed withdrawal with insufficient gas. After L1 tx fails, card should go back to allowing you to try again.

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:
